cmd/gogio/androidbuild.go

```diff
@@ -548,43 +548,67 @@ func signAPK(tmpDir string, tools *androidTools, bi *buildInfo) error {
 	if err != nil {
 		return err
 	}
+
+	if bi.key == "" {
+		if err := defaultAndroidKeystore(tmpDir, bi); err != nil {
+			return err
+		}
+	}
+
+	buildParams := []string{"sign"}
+	if bi.cert == "" {
+		// Keystore.
+		buildParams = append(buildParams, []string{
+			"--ks-pass", "pass:" + bi.password,
+			"--ks", bi.key,
+		}...)
+	} else {
+		// Cert + Key.
+		buildParams = append(buildParams, []string{
+			"--key-pass", "pass:" + bi.password,
+			"--key", bi.key,
+			"--cert", bi.cert,
+		}...)
+	}
+
+	_, err = runCmd(exec.Command(filepath.Join(tools.buildtools, "apksigner"), append(buildParams, apkFile)...))
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func defaultAndroidKeystore(tmpDir string, bi *buildInfo) error {
```

Comment:
I'm trying to add the "Android App Bundle", which may be the my next PR. The App Bundle uses `jarsigner` instead of `apksigner`, so the `defaultAndroidKeystore` function can be re-used.